### PR TITLE
docs: add etcd-ocs.md

### PR DESCRIPTION
**Pull request by zbwright**

### CODE REVIEW

File: Documentation/alm/etcd-ocs.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* **Backups included:** Trigger backups to an object store as part of disaster recovery planning.
+
+## Deploying etcd OCS
+
+Use Tectonic Console to enable the etcd OCS for selected namespaces. By default, the etcd Open Cloud Service will deploy a 3 member cluster, which may be resized or updated by editing its YAML manifest.
+
+Objects created using the etcd OCS will be labeled `app=etcd` and `etcd_cluster=<cluster-name>`.
+
+Using the etcd Open Cloud Service to deploy an etcd cluster will create the following Kubernetes objects:
+* An etcd CRD
```
Comment (robszumski) [inline Documentation/alm/etcd-ocs.md]: I am new to advanced Kubernetes. What is a CRD?

Comment (robszumski) [reply Documentation/alm/etcd-ocs.md]: do we even need to include that? maybe just call it an object?

File: Documentation/alm/etcd-ocs.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* 3 Pod etcd cluster
+* 2 Services, one for internal communication and one for use with etcdctl/API
+
+## Working with Kubernetes Services
+
+For every etcd cluster created Tectonic will create an etcd client service in the same namespace with the name `<cluster-name>-client`.
+
+The client service is of type `ClusterIP` and is accessible only from within the Kubernetes cluster's network.
```
Comment (robszumski) [inline Documentation/alm/etcd-ocs.md]: If you'd like to expose this address outside of the cluster, a new service of type LoadBalancer or NodePort can be created.

File: Documentation/alm/etcd-ocs.md (hunk 1 above)
Comment (zbwright) [reply Documentation/alm/etcd-ocs.md]: Calling it out makes it easier for users to find it in Console - I'll expand the acronym.

**Pull request merged**